src/lib/parse_org.js

```diff
@@ -598,6 +598,13 @@ const parseLogbook = rawText => {
   };
 };
 
+// TODO: Active timestamps in the headline, and (at least) active
```

Comment:
In Org mode, an active timestamp anywhere in the description of the headline will add it to the agenda, for example:

```
*** TODO Foo
    :PROPERTIES:
    :foo:      bar
    :END:

<2019-12-28 Sat>
```

That includes adding it to the headline, but it's likely seldomly done.

```
*** TODO <2019-12-28 Sat> Foo
```

Reply:
Actually, my task lists for work and private look all like your second example ^^

That has to do with the inferior vim plugin for orgmode but I also prefer the slimmer syntax.